tests/http_apis/conf/diff_config.toml

```diff
@@ -0,0 +1,40 @@
+# diff Configuration.
+
+log-level = "info"
+
+chunk-size = 10
+
+check-thread-count = 4
+
+sample-percent = 100
+
+use-rowid = false
+
+use-checksum = true
+
+fix-sql-file = "fix.sql"
+
+# tables need to check.
+[[check-tables]]
+schema = "http_apis"
+tables = ["~.*"]
+
+[[source-db]]
+host = "127.0.0.1"
+port = 3306
+user = "root"
+password = ""
+instance-id = "source-1"
+
+[[source-db]]
```

Comment:
this db is not used in this test

Reply:
https://github.com/pingcap/dm/pull/224/commits/605187a5b75a9401cd164d7ebdb012dac35efc2d